### PR TITLE
[CSLA-62] Avoid long target string in Windows shortcut

### PR DESCRIPTION
**Pull request by jmitchell**
Replaces PR #185.

### CODE REVIEW

File: installers/WindowsInstaller.hs [hunk 1]
```diff
@@ -11,21 +11,30 @@ import           Turtle             (echo, proc, procs)
...
-  , installerPath = "%APPDATA%\\Daedalus\\Installer.exe"
-  , runtimePath = "%APPDATA%\\Daedalus\\"
-  , reportServerUrl = "http://35.156.164.19:8080"
-  , systemStart = Nothing
-  }
+launcherScript :: String
+launcherScript = unlines
+  [ "@echo off"
+  , ""
+  , "powershell.exe -Command \"(Get-ItemProperty HKLM:\\SOFTWARE\\Wow6432Node\\Daedalus).Install_Dir\" > install_dir.txt"
```
Comment (georgeee) [inline installers/WindowsInstaller.hs]: Are you sure PowerShell is present in all WIndows 7+?

Comment (jmitchell) [inline installers/WindowsInstaller.hs]: Great point. I'll find a more portable solution.

**Comment by jmitchell** [comment]
@georgeee: I verified this installs, runs, and uninstalls properly in Win Server 2012 r2, both to default and non-default directories. Do you mind doing some additional testing on other Windows versions?

**Comment by fersel** [comment]
@jmitchell I tested it and though this works fine using the shortcut I don't think the current approach is good enough. Two main things I'm concerned about:
1) putting .bat file in %appdata%. It should stay in the installation folder
2) not writing down full paths in the .bat file. That makes us depend on the shortcut (which might be deleted or whatever), i.e. if you try to launch via .bat directly that *won't* work.
(these problems are somewhat connected)

Also:
* each version installation will override the prev version shortcut (you can't change that during the installation process), that's inconvenient.
* can we have different folders in %appdata% for different app versions as well?

I can describe this more detailed if needed, but I'd prefer to do that in the ticket.

**Comment by jmitchell** [comment]
@fersel, I've responded in the ticket. Thank you for testing it and giving your feedback.

**Comment by domenkozar** [comment]
You should now make a PR against master, since some parts changed a bit. We will be rolling out 0.4 soon so that train is gone :)

**Pull request merged**